cmd/definitions/tmpl/operation.tmpl

```diff
@@ -20,7 +20,33 @@ type {{ $i.DisplayName }} interface {
     {{ $op.Name | toPascal }}WithContext(ctx context.Context,{{ $op.FormatParams }}) ({{ $op.FormatResultsWithPackageName "storage" }})
     {{ end }}
     {{ end }}
+
+    mustEmbedUnimplemented{{ $i.DisplayName }}()
+}
+
+// Unimplemented{{ $i.DisplayName }} must be embedded to have forward compatible implementations.
+type Unimplemented{{ $i.DisplayName }} struct {}
+
+func (s Unimplemented{{ $i.DisplayName }}) mustEmbedUnimplemented{{ $i.DisplayName }}() {}
+
+func (s Unimplemented{{ $i.DisplayName }}) String() string {
+    return "Unimplemented{{ $i.DisplayName }}"
+}
+
+{{ range $_, $op := $i.Ops }}
+func (s Unimplemented{{ $i.DisplayName }})  {{ $op.Name | toPascal }}({{ $op.FormatParams }}) ({{ $op.FormatResultsWithPackageName "storage" }}) {
+    {{- if not $op.Local }}
+    err = NewOperationNotImplementedError("{{ $op.Name }}")
+    {{- end }}
+    return
+}
+{{- if not $op.Local }}
+func (s Unimplemented{{ $i.DisplayName }}) {{ $op.Name | toPascal }}WithContext(ctx context.Context,{{ $op.FormatParams }}) ({{ $op.FormatResultsWithPackageName "storage" }}) {
+    err = NewOperationNotImplementedError("{{ $op.Name }}")
```

Comment:
add `WithContext` after `op.Name`?

Reply:
`op` is the operation instead of function name, and those functions (WithCtx and NoCtx version) are auto generated so they should not be missing.